src/Components/Web.JS/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "removeComments": false,
     "sourceMap": true,
     "target": "es2019",
+    "module": "es2020",
```

Comment:
Out of interest, what's changed to require this?

Reply:
I was getting some complaints with the dynamic import, and doesn't hurt to update it.

Reply:
Could you say which specific API feature it was complaining about? Would be good to check we're not losing compat with something like a recent version of iOS. I know we *say* we support "current" browsers but we still need to reason about times that we explicitly drop support for a major class of browsers.

Reply:
TS1323: Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'esnext', 'commonjs', 'amd', 'system', or 'umd'.

Reply:
Thanks, that explains it. I didn't understand why we had new requirements given that we already called `import` dynamically for the existing JS module loading feature. But I now see that `Microsoft.JSInterop`'s `tsconfig.json` includes:

```json
"module": "ESNext",
```